src/popup/components/MultisigVaultCreateReview.vue

```diff
@@ -208,8 +208,9 @@ export default defineComponent({
   gap: var(--gap);
 
   .creator-error-message {
-    @extend %face-sans-14-regular;
```

Comment:
But in figma the size is `14px` for this element.

Reply:
14 px is only for amount value
![image](https://github.com/aeternity/superhero-wallet/assets/90620504/03407fcc-9170-4b64-9e4c-f6d9aa5584dc)

Reply:
fixed